### PR TITLE
Include sbti file

### PR DESCRIPTION
**Pull request by AlexAxthelm**
Note that this targets #331

### CODE REVIEW

**Comment by AlexAxthelm** [comment]
@Antoine-Lalechere @MonikaFu I'm seeing an error for pensionfund ES (and it's the only one 🎉 ), but I'm hoping that one of you two might have better information on how to resolve it.

https://github.com/RMI-PACTA/workflow.transition.monitor/actions/runs/9749207788/job/26907559916#step:10:104
```
Quitting from lines 192-232 [scorecard] (./scorecard.Rmd)
Error in `pull()`:
! Problem while evaluating `peergroup`.
Caused by error in `.data$peergroup`:
! Column `peergroup` not found in `.data`.
Backtrace:
  1. ... %>% pull(.data$peergroup)
 16. rlang:::abort_data_pronoun(x, call = y)
Execution halted
```

**Comment by MonikaFu** [comment]
@AlexAxthelm I updated the testing files in https://github.com/RMI-PACTA/user_results/pull/24#issuecomment-2202295578 which should resolve the issue. It seems that the first testing files provided by @Antoine-Lalechere did not contain all peer groups. I also discovered that the `number_peers.csv` file did not contain all peer groups. It seems not to cause issues in tests but I updated it just in case.

File: DESCRIPTION [hunk 1]
```diff
@@ -47,7 +47,7 @@ Imports:
     rmarkdown, 
     shiny
 Remotes: 
-    RMI-PACTA/pacta.executive.summary, 
+    RMI-PACTA/pacta.executive.summary@update-sbti-section-scorecard,
```
Comment (cjyetman) [inline DESCRIPTION:50]: I don't think we should be targeting a PR branch on main?

Comment (MonikaFu) [reply DESCRIPTION:50]: this is for testing purposes I believe. And it is not going to be merged into main

Comment (cjyetman) [reply DESCRIPTION:50]: ok, then can we turn it back into a draft, mark it clearly as such in the title, and not request reviews from people?

Comment (jdhoffa) [reply DESCRIPTION:50]: @cjyetman This PR isn't targeting `main`, it's targeting `update-executive-summary` which is itself is a PR branch. 

This whole PR is just a glorified PR commit

**Review by jdhoffa — APPROVED** [review]

**Comment by jdhoffa** [comment]
Given that this is merging into #331 and not `main` this LGTM. 

I will do a more in-depth review of #331

**Pull request merged**